### PR TITLE
Init build-push to quay/gatekeeper

### PR DESCRIPTION
**Pull request by yiraeChristineKim**
**What this PR does / why we need it**:

**Which issue(s) this PR fixes** *(optional, using `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when the PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

### CODE REVIEW

**Comment by yiraeChristineKim** [comment]
/override "build and push to quay / Image build and push"

File: .github/workflows/build-push-stolostron.yaml [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+jobs:
+  build:
+    name: Image build and push
+    runs-on: ubuntu-latest  
+
+    steps:
+    - uses: actions/checkout@v4
+      with: 
+        fetch-depth: 0 # Fetch all history for all tags and branches
```
Comment (dhaiducek) [inline .github/workflows/build-push-stolostron.yaml:21]: nit: Fetching everything isn't really necessary

```suggestion
```

File: .github/workflows/build-push-stolostron.yaml [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+    - uses: docker/login-action@v2
+      with:
+        registry: quay.io
+        username: ${{ secrets.QUAY_USER }}
+        password: ${{ secrets.QUAY_PASSWORD }}
+
+    - name: build and push
+      run: |
+        REPOSITORY=${{env.REPOSITORY}} VERSION=${{env.VERSION_TAG}} make docker-buildx-release
```
Comment (dhaiducek) [inline .github/workflows/build-push-stolostron.yaml:31]: nit: Missing trailing newline
```suggestion
        REPOSITORY=${{env.REPOSITORY}} VERSION=${{env.VERSION_TAG}} make docker-buildx-release 

```

**Review by dhaiducek — COMMENTED** [review]
A couple nits, but this seems like it'll work 🙂

File: .github/workflows/build-push-stolostron.yaml [hunk 3]
```diff
@@ -0,0 +1,30 @@
+name: build and push to quay
+
+on:
+  workflow_dispatch:
```
Comment (dhaiducek) [inline .github/workflows/build-push-stolostron.yaml:4]: Sorry, I missed this the first time. We don't want this workflow to be run on-demand.

```suggestion
```

File: .github/workflows/build-push-stolostron.yaml [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+    - uses: docker/login-action@v2
+      with:
+        registry: quay.io
+        username: ${{ secrets.QUAY_USER }}
+        password: ${{ secrets.QUAY_PASSWORD }}
+
+    - name: build and push
+      run: |
+        REPOSITORY=${{env.REPOSITORY}} VERSION=${{env.VERSION_TAG}} make docker-buildx-release
+
```
Comment (dhaiducek) [inline .github/workflows/build-push-stolostron.yaml:30]: nit: trailing whitespace after the newline 🙂 
```suggestion

```

**Review by dhaiducek — CHANGES_REQUESTED** [review]

**Review by dhaiducek — APPROVED** [review]

**Pull request closed without merging**